components/camel-gson/src/test/java/org/apache/camel/component/gson/GsonMarshalExclusionTest.java

```diff
@@ -43,7 +44,9 @@ public void testMarshalAndUnmarshalPojoWithExclusion() throws Exception {
 
         Object marshalled = template.requestBody("direct:inPojoExcludeWeight", in);
         String marshalledAsString = context.getTypeConverter().convertTo(String.class, marshalled);
-        assertEquals("{\"age\":30,\"height\":190}", marshalledAsString);
+        assertTrue(marshalledAsString.contains("\"height\":190"));
+        assertTrue(marshalledAsString.contains("\"age\":30"));
+		assertFalse(marshalledAsString.contains("\"weight\":70"));
```

Comment:
This line needs to indent.

Reply:
just run mvn install -P format to reformat

Reply:
Okay, Doing that!